src/main/java/com/plesba/datamanager/DataManager.java

```diff
@@ -99,8 +106,8 @@ public void run() {
             }
         } else if (datasource.equals( "csv")) {
             //csvreader - read from csv file / write to output stream
-            LOG.info("DataManager input from csv file: " + dataMgrProps.getProperty("csv.infilename"));
             csvSource = new CSVSource(dataMgrProps.getProperty("csv.infilename"), outputStream1);
+            LOG.info("DataManager input from csv file: " + csvSource);
```

Comment:
Does 'CSVSource' have an overridden toString() method? Or what is produced in the log when line 110 executes? This line's argument concatenates a literal string with an object.